chore: update meta tags on blog posts

paulatulis opened the pull request:
Test to see if including url in meta will correct the rest of the og tags

lottamus commented:
Deploy preview for *stoplightio* ready!

Built with commit a761e074ffeb4ccfbe76b19e24ba3b7c98f75ed9

https://deploy-preview-1341--stoplightio.netlify.com

paulatulis commented:
@ksatx would you be able to approve this so I can test the production URL with the new meta tags? Removed og and moved url to root meta object per this [comment](https://github.com/stoplightio/stoplight.io/issues/1330#issuecomment-581773086).

lottamus commented:
Deploy preview for *stoplightio* ready!

Built with commit f9ca0b6fa6e6843ac458e7c02c8ca18d1f799fe6

https://deploy-preview-1341--stoplightio.netlify.com
